test/js-native-api/test_error/test.js

```diff
@@ -60,24 +60,16 @@ assert.throws(() => {
   test_error.throwTypeError();
 }, /^TypeError: type error$/);
 
-function testThrowArbitrary(value) {
-  assert.throws(
-    () => test_error.throwArbitrary(value),
-    (err) => {
-      assert.strictEqual(err, value);
-      return true;
-    });
-}
-
-testThrowArbitrary(42);
-testThrowArbitrary({});
-testThrowArbitrary([]);
-testThrowArbitrary(Symbol('xyzzy'));
-testThrowArbitrary(true);
-testThrowArbitrary('ball');
-testThrowArbitrary(undefined);
-testThrowArbitrary(null);
-testThrowArbitrary(NaN);
+const testThrowArbitrary = (value) => assert.throws(
```

Comment:
Nit: any reason for using a function expression? I find the original declaration easier to read.

Reply:
nice idea, resolved